Test/allocated1/Allocated1.dfy

```diff
@@ -1,4 +1,4 @@
 
-// RUN: %exits-with 4 %dafny /verifyAllModules /allocated:1 /errorLimit:99 /compile:0 /dprint:"%t.dprint" /autoTriggers:0 "%s" > "%t"
+// RUN: %exits-with 4 %dafny /verifyAllModules /allocated:1 /errorLimit:0 /compile:0 /dprint:"%t.dprint" /autoTriggers:0 "%s" > "%t"
```

Comment:
This might make sense to include with 3.x, too. Same with other instances of `/errorLimit:0`.